staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -445,6 +445,7 @@ message PodIP{
     string ip = 1;
 }
 // PodSandboxNetworkStatus is the status of the network for a PodSandbox.
+// Ignored for pods sharing the host networking namespace.
```

Comment:
Is there a chance we might add new fields in the future that it wouldn't make sense to ignore for hostNetwork pods? Maybe hedge a little bit here and prepend "Currently" ?

Reply:
great point